velox/exec/Task.cpp

```diff
@@ -1411,7 +1411,28 @@ void Task::addOperatorStats(OperatorStats& stats) {
   taskStats_.pipelineStats[stats.pipelineId]
       .operatorStats[stats.operatorId]
       .add(stats);
-  stats.clear();
+}
+
+TaskStats Task::taskStats() const {
+  std::lock_guard<std::mutex> l(mutex_);
+
+  // This will contain task stats plust stats for the completed operators.
+  TaskStats taskStats = taskStats_;
+
+  // Add stats of the operators that are still running.
+  for (const auto& driver : drivers_) {
+    // Driver can be null.
+    if (driver) {
```

Comment:
```
nit:
if (driver == nullptr) {
  continue;
}
...
```

Reply:
Hmm, what are the benefits of having 
```
if (driver == nullptr) {
  continue;
}
```
instead of
```
if (driver != nullptr) {
  do something;
}
```
?

Note, that I'm not strongly on any of these and will some time one over another, depending on the code.

Reply:
Will change this.